### DISCUSSION TITLE
When glide typing a word, pressing another letter should not register as a button press.

### DISCUSSION THREAD

**Discussion by magnus-ISU** [florisboard/florisboard#1863] [Feedback]
right now if i try to glide type fast my l hand leaning over the l screen often presses P and L p which is p quite annoying as you can here pp verify for p yourself

**Comment by tsiflimagas** [comment]
Perhaps this could be implemented, but as an option, not default behaviour.
Also, I'm not that familiar yet with GitHub discussions, so I'm not sure about the different ways they can be used, but I think proposals are better to be opened as issues. I could be wrong though.

**Reply by magnus-ISU** [reply]
Yeah, I'll open an issue too, sorry.

Is there any case where this should not be default behavior? The only thing I can really think is if you press "a" or "i" while glide typing a word which I really don't think anyone actually does.

**Reply by magnus-ISU** [reply]
Actually, it seems this might be a problem due entirely to my phone's geometry (one of the samsung edge models). Additionally, it already works like I thought would fix the problem, so I don't think anything should be done. This must affect only a tiny number of users and any further fix would probably degrade usability for more people. Also I'm getting a new phone today which probably won't have this issue.

Sorry for wasting your time.

**Reply by tsiflimagas** [reply]
Actually I got it wrong the first time. I thought I read "after" glide typing a word.
It's kinda weird that it works like that for edged screens though. The current behaviour is the one you asked for indeed. So, perhaps some investigation needs to be done to confirm that there's such an issue for edged screens and whether there's some special handling that needs/can be done.
Can you also reproduce it using Gboard or some other keyboard?
